tools/f18/f18.cpp

```diff
@@ -308,6 +311,15 @@ std::string CompileFortran(std::string path, Fortran::parser::Options options,
         nullptr /* action before each statement */, &asFortran);
     return {};
   }
+  if (driver.dumpPreFirTree) {
+    if (auto ast{Fortran::lower::createPFT(parseTree)}) {
+      Fortran::lower::annotateControl(*ast);
+      Fortran::lower::dumpPFT(llvm::outs(), *ast);
+    } else {
+      std::cerr << "Pre FIR Tree is NULL.\n";
```

Comment:
llvm::errs() vs. std::cerr?

Reply:
Ah, actually ~ so so, `std::cerr` is already used in this f18 file, I do not want to spearhead the change to llvm stream here (I actually tried to hide the `llvm::outs()` to avoid troubles here, but I then hit llvm runtime issues (I think the driver should initialized it or something) so I left it that way).

Reply:
That makes sense. +1